confgenerator/agentmetrics.go

```diff
@@ -297,24 +289,39 @@ func (r AgentSelfMetrics) LoggingMetricsPipelineProcessors() []otel.Component {
 		// DeltaToCumulative keeps in memory information of previous delta points
 		// to generate a valid cumulative monotonic metric.
 		otel.DeltaToCumulative(),
+		otel.MetricStartTime(),
```

Comment:
I'm a bit concerned about that this may change the `LoggingMetrics` some unintended way. Some requests : 

- Could we please separate all `MetricStartTime()` changes into a separate PR to makes sure they work individually with the GCM exporter ?
- Could we document and track the effort to fix the upstream issue with `int64` metrics in the `MetricStartTime` processor ?